docs/index.md

```diff
@@ -42,9 +42,14 @@ download and install the latest Theme Kit for you.
 curl -s https://shopify.github.io/themekit/scripts/install.py | sudo python
 ```
 
-### Arch installation
+### Community Packages
 
-There is a package available for install on the [AUR](https://aur.archlinux.org/packages/shopify-themekit-bin)
+Theme Kit is available through other package manager distributions, but as these are not
```

Comment:
Even though we only list one, I wrote this is as plural since there may be others we're not aware of.